nc.js

```diff
@@ -0,0 +1,28 @@
+const fs = require('fs')
+const name = process.argv[2]
+if (!name) {
+  console.error('You must specify a component name!')
+  process.exit(1)
+}
+fs.mkdirSync(`src/components/${name}`)
+fs.writeFileSync(`src/components/${name}/index.tsx`, `
+import React from 'react'
+import styles from './${name}.module.css'
+
+export default () => (
+  <div className={styles.container}>
+
+  </div>
+)
+`.trim())
+fs.writeFileSync(`src/components/${name}/${name}.module.css`, `
+.container {
+  
+}
+`.trim())
+const compIndex = fs.readFileSync('src/components/index.ts')
```

Comment:
Note sure if this complies with your linting tools, but I would prefer if there was better space distribution within this file.

Reply:
Got it! WIll fix :D

Reply:
I made that in like 10 seconds lol